tests/unit/test_charm.py

```diff
@@ -175,15 +219,15 @@ def test_update_container_config(harness: Harness, mocked_sql_migration: MagicMo
 
 def test_on_config_changed_without_service(harness: Harness) -> None:
     setup_postgres_relation(harness)
-    harness.update_config({"login_ui_url": "http://some-url"})
+    setup_login_ui_relation(harness)
 
     assert harness.charm.unit.status == WaitingStatus("Waiting to connect to Hydra container")
 
 
 def test_on_config_changed_without_database(harness: Harness) -> None:
     harness.set_can_connect(CONTAINER_NAME, True)
     harness.charm.on.hydra_pebble_ready.emit(CONTAINER_NAME)
-    harness.update_config({"login_ui_url": "http://some-url"})
+    setup_login_ui_relation(harness)
```

Comment:
Similar comment as above, let's keep the test focused on the missing db relation
```suggestion

```